### PR TITLE
Adding several improvements to readme

### PR DESCRIPTION
**Pull request by theburi**
Several Changes to normalize and improve this repository to fix several inconsistencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,25 +66,26 @@ Keycloak is used to manage users. Here you can log in with the user `admin` and
...
 
 ```
 docker compose -f docker-compose-core.yaml up -d
 ```
+
 ### Deploying BPMN diagrams
 
 In addition to the local environment setup with docker compose, use the [Camunda Desktop Modeler](#desktop-modeler) to locally model BPMN diagrams for execution and directly deploy them to your local environment.
-As an enterprise customer, you can [use Web Modeler](#web-modeler-self-managed).
+You can also [use Web Modeler](#web-modeler-self-managed).
```
Comment (HanselIdes) [inline README.md:88]: Thanks for cleaning this up

File: README.md [hunk 2]
```diff
@@ -24,32 +32,25 @@ For production setups we recommend using [Helm charts](https://docs.camunda.io/d
...
-- Connectors
-- Optimize
-- Identity
-- Elasticsearch
-- Keycloak
-- PostgreSQL
+To spin up a  Camunda Platform 8 Self-Managed environment locally you can use the following docker compose configuration files:
+ - [docker-compose.yaml](docker-compose.yaml) file contains all Camunda 8 Components for a full stack deployment: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity, Elasticsearch, Keycloak, Web Modeler, PostgreSQL
+ - [docker-compose-core.yaml](docker-compose.yaml) file contains  Camunda 8 Orchestration cluster components: Zeebe, Tasklist, Operate, Optimize, Identity and Connectors
+ - [docker-compose-web-modeler.yaml](docker-compose.yaml) file contains Camunda 8 Web Modeler standalone installation
```
Comment (HanselIdes) [inline README.md:45]:
```suggestion
 - [docker-compose-web-modeler.yaml](docker-compose.yaml) file contains Camunda 8 Web Modeler for modeling without Play or an orchestration cluster
```

**Review by HanselIdes — CHANGES_REQUESTED** [review]
I have a subtle change to show the limitations of the web modeler config option

**Review by hisImminence — APPROVED** [review]
lgtm

File: README.md [hunk 3]
```diff
@@ -33,35 +33,33 @@ For production setups we recommend using [Helm charts](https://docs.camunda.io/d
...
-Now you can navigate to the different web apps and log in with the user `demo` and password `demo`:
+Navigate to the different web apps and log in with the user `demo` and password `demo`:
 - Operate: [http://localhost:8081](http://localhost:8081)
 - Tasklist: [http://localhost:8082](http://localhost:8082)
 - Optimize: [http://localhost:8083](http://localhost:8083)
 - Identity: [http://localhost:8084](http://localhost:8084)
 - Elasticsearch: [http://localhost:9200](http://localhost:9200)
 
-Keycloak is used to manage users. Here you can log in with the user `admin` and password `admin`
+Log in to Keycloak using user: `admin` and password: `admin` to manager users.
```
Comment (HanselIdes) [inline README.md:62]: Minor UX comment: is there a technical need to have separate passwords between keycloak and camunda? People already have dozens of passwords from other accounts

Comment (HanselIdes) [inline README.md:62]:
```suggestion
Log in to Keycloak using user: `admin` and password: `admin` to manage users.
```
typo

Comment (hisImminence) [reply README.md]: Good point - Lets make this a follow up question to unblock this PR!

Comment (hisImminence) [reply README.md]: --> https://github.com/camunda/camunda-platform/issues/1085

**Review by HanselIdes — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (HanselIdes) [reply README.md]: So there's no technical need, but you recommend not changing it. OK

**Pull request merged**